src/api/order/order.resolver.ts

```diff
@@ -0,0 +1,26 @@
+import { Resolver, Query, Args, Mutation } from '@nestjs/graphql';
+import { Order, OrderSelect } from './model';
+import { OrderArgs, OrderCreateInput } from './dto';
+import { OrderService } from './order.service';
+import { GraphQLFields, IGraphQLFields } from '../../shared/decorators';
```

Comment:
This should be takne from 'shared/decorators' meaning using absolute path not relative.